ehsm_kms_service/enroll_app/main.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (C) 2021-2022 Intel Corporation
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ *   1. Redistributions of source code must retain the above copyright
+ *      notice, this list of conditions and the following disclaimer.
+ *   2. Redistributions in binary form must reproduce the above copyright
+ *      notice, this list of conditions and the following disclaimer in
+ *      the documentation and/or other materials provided with the
+ *      distribution.
+ *   3. Neither the name of Intel Corporation nor the names of its
+ *      contributors may be used to endorse or promote products derived
+ *      from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY CLEANUP OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+using namespace std;
+
+#include <fstream>
+
+#include "params.h"
+#include "log_utils.h"
+#include "rest_utils.h"
+
+
+int main(int argc, char* argv[])
+{    
+    print_d("***enroll app start.");
+    enroll_status_t ret = ENL_OK;
+    RetJsonObj retJsonObj;
+
+    std::string msg0_str;
+    std::string msg2_str;
+    std::string msg4_str;
+
+    // only one parameter, it is ehsm_kms_url
+    std::string ehsm_kms_url = argv[1];
+    if (ehsm_kms_url.empty())
+    {
+        print_e("ehsm_kms_url undefined.");
+        ret = ENL_CONFIG_INVALID;
+        goto OUT;
+    }
+    print_d("ehsm_kms_url : %s", ehsm_kms_url.c_str());
+
+    print_d("=> First handle send msg0,return msg1.");
+    ret = ra_get_msg0(&msg0_str);
+    print_d("msg0 : \n%s", msg0_str.c_str());
+
+    print_d("post RA_HANDSHAKE_MSG0.....");
+    post_KMS(ehsm_kms_url + "?Action=RA_HANDSHAKE_MSG0", msg0_str, &retJsonObj);
```

Comment:
remove the unnecessary debug logs.

Reply:
print_d It is controlled by the [#define IS_DEBUG false] constant and is currently set to false No output

Reply:
yes, i know.
you only need to use print_d for the necessary msg, remove them even it won't print out.